frontend/public/components/namespace.jsx

```diff
@@ -270,7 +270,7 @@ export const TopPodsBarChart = ({ns}) => (
   <Bar
     title="Memory Usage by Pod (Top 10)"
     namespace={ns.metadata.name}
-    query={`sort_desc(topk(10, sum by (pod_name)(container_memory_usage_bytes{container_name!="POD",container_name!="",pod_name!="", namespace="${ns.metadata.name}"})))`}
+    query={`sort_desc(topk(10, sum by (pod_name)(container_memory_working_set_bytes{container_name!="POD",container_name!="",pod_name!="", namespace="${ns.metadata.name}"})))`}
```

Comment:
we're looking for the topk pod right? in that case we should filter with `container=""`, so we get the pod cgroup hierarchy